docker-compose.dev.yml

```diff
@@ -4,6 +4,7 @@
 services:
   app:
     # Add --reload to command, to reload on file change.
+    # command: ["fil-profile", "python", "-m",  "gunicorn", "-c", "conbench/gunicorn-conf.py", "--reload"]
```

Comment:
Note: the fact that this works isn't obvious at all. It's not like I just copy/pasted a command from some docs here.

- It wasn't obvious that gunicorn can be invoked via `python -m gunicorn ..`
- It wasn't obvious that we can wrap this entire thing in to `fil-profile`
- It wasn't obvious that `fil-profile` manages to keep track of the _child process_ spawned by this machinery; the first process that's spawned here is the gunicorn master process, and the work happens in a worker process. This entry point here ensures that even the child is instrumented properly (with fil-profile specifics), and it needs to be combined with the file-profile API-based tracking (i.e. usage of the `filprofile()` function in code)